src/settings.ts

```diff
@@ -53,6 +53,7 @@ module powerbi.extensibility.visual.settings {
 
     export class LabelsSettings {
         show: boolean = true;
+        displayAll: boolean = true;
```

Comment:
Please do not use "true" value by default for your setting cause this can affect to already created users reports